wifiphisher/common/accesspoint.py

```diff
@@ -151,3 +155,43 @@ def on_exit(self):
         # sleep 2 seconds to wait all the hostapd process is
         # killed
         time.sleep(2)
+
+    def init_thread_read_victims_file(self):
+        """
+        Starts background thread that reads dnsmasq file with clients connected
+        """
+
+        thread = threading.Thread(target=self.read_connected_victims_file)
+        thread.daemon = True
+        thread.start()
+
+    def read_connected_victims_file(self):
+        """
+        Reads dnsmasq.leases file and creates and inserts Victim
+        objects to the dictionary
+        """
+
+        while True:
+            if os.path.isfile('/var/lib/misc/dnsmasq.leases'):
+                with open("/var/lib/misc/dnsmasq.leases", "r") as dnsmasq_leases:
+                    for line in dnsmasq_leases:
+                        line = line.split()
+                        if len(line) > 0:
+                            mac_address = line[1].strip()
+                            ip_address = line[2].strip()
+                            # Get instance of victims dic
+                            victims_instance = victim.Victims.getInstance()
+                            if mac_address in victims_instance.victims_dic.keys():
```

Comment:
actually here you only need:
`if mac_address in victims_dict` it will be faster than `if mac_address in victims_dic.keys()`
When you use `keys()`, this makes the keys become as list and this will make the runtime from O(1) to O(n)